tools/stonith_admin.c

```diff
@@ -98,7 +105,6 @@ static struct crm_option long_options[] = {
     {   "history", required_argument, NULL, 'H',
         "Show last successful fencing operation for named node\n"
         "\t\t\t(or '*' for all nodes). Optional: --timeout, --cleanup,\n"
-        "\t\t\t--quiet (show only the operation's epoch timestamp),\n"
```

Comment:
This is an example where --quiet reduces detail rather than silence output altogether. So I'm afraid this particular tool (at least with this one option) won't be as simple as quiet == /dev/null, though are definitely others that will be.